recipes-kernel/linux/files/5.x-kernel/0010-dts-add-the-cp2102n-reset-pin-defination.patch

```diff
@@ -0,0 +1,54 @@
+From d5c4ed18c01d4f42e09c0cf6b76083d3f6a38471 Mon Sep 17 00:00:00 2001
+From: Chao Zeng <chao.zeng@siemens.com>
+Date: Mon, 22 Mar 2021 13:34:18 +0800
+Subject: [PATCH 10/14] dts : add the cp2102n reset pin defination
+
+configure the cp2102n reset pin and named lines to control cp2102n reset function
```

Comment:
Is that PG2-specific, or were we missing that for PG1 as well? And I thought there is only a cp2102 on the Advanced variant, no?

Reply:
yes, only for the PG2 advanced board.